package.json

```diff
@@ -22,11 +22,11 @@
     "node": ">=4"
   },
   "scripts": {
-    "test": "mocha --check-leaks -R spec",
+    "test": "mocha --check-leaks -R spec test/*-tests.js",
     "posttest": "npm run lint",
-    "test:coverage": "istanbul cover _mocha -- --check-leaks test/*-tests.js",
+    "test:coverage": "istanbul cover --print none --report none --include-pid _mocha -- --check-leaks test/*-tests.js && istanbul report text-summary lcov && rm -f test/coverage/coverage-*.json",
```

Comment:
@elyscape I'm coming back yo this little point: from what you are explaining above, no `coverage-*.json` files shall be present before the test coverage starts. Instead of a `posttest:coverage`, that calls for a `pretest:coverage` script, doesn't it? That way using a `pretest:coverage` script, the needed purpose of the cleaning would be better advertized, wouldn't it?